os/sdl/gl_config.c

```diff
@@ -0,0 +1,8 @@
+#include "common.h"
+#include "gl_common.h"
+#include "gl_config.h"
+#include "gl_hints.h"
+
+cen64_gl_config *cen64_gl_config_create(cen64_gl_display display,
+  cen64_gl_screen screen, const cen64_gl_hints *hints, int *matching) {
```

Comment:
For the sake of parity with the other backends, do you think you could an implementation setting attributes similarly to the X11 one, but with SDL_GL_SetAttribute calls? You can skip attributes that doesn't map to the ones supported by SDL_GL_SetAttribute.